scripts/gen_config.py

```diff
@@ -1,223 +0,0 @@
-# coding=utf-8
```

Comment:
just to be sure, this utility only generates config files from some file generated by rho?

Reply:
I'm going to keep this script for now and don't remove it.